src/Controller.js

```diff
@@ -0,0 +1,91 @@
+import { NAME_DELIMITER, MOVE_NUMBER } from './constants/index.js';
+import Car from './models/Car.js';
+import Console from './utils/Console.js';
+import randomNumberInRange from './utils/RandomNumberInRange.js';
+import Validator from './utils/Validator.js';
+import View from './View.js';
+
+class Controller {
+  #cars;
+
+  constructor() {
+    this.#cars = [];
+  }
+
+  async play() {
+    await this.#initCars();
+    this.#raceStart(await this.#initCount());
+    this.#presentWinner();
+    Console.close();
+  }
+
+  async #initCars() {
+    View.naming();
+    const names = await this.#readNames();
+    names.forEach((name) => {
+      this.#cars.push(new Car(name));
+    });
```

Comment:
Array.prototype.map 을 사용하면 어떻게 더 깔끔하게 리팩토링할 수 있을까요? 😄

Reply:
```suggestion
    this.#cars = names.map((name) => new Car(name));
```
이런식으로 가능할 것 같습니다! 확실히 더 보기 좋다고 느껴집니다.